app/javascript/dashboard/routes/dashboard/settings/inbox/Settings.vue

```diff
@@ -102,21 +102,38 @@
             }}
           </p>
         </label>
-
-        <woot-input
-          v-if="greetingEnabled"
-          v-model.trim="greetingMessage"
-          class="medium-9 columns"
-          :label="
-            $t('INBOX_MGMT.ADD.WEBSITE_CHANNEL.CHANNEL_GREETING_MESSAGE.LABEL')
-          "
-          :placeholder="
-            $t(
-              'INBOX_MGMT.ADD.WEBSITE_CHANNEL.CHANNEL_GREETING_MESSAGE.PLACEHOLDER'
-            )
-          "
-        />
-
+        <section v-if="greetingEnabled">
+          <textarea
```

Comment:
Instead of textarea should we use `import ResizableTextArea from 'shared/components/ResizableTextArea';` which is used in `app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue` ?

Reply:
Replaced with `<resizable-text-area>` component in [commit](https://github.com/chatwoot/chatwoot/pull/2952/commits/405630308096be71015daa6f7aa307803a3b50b4)